libdevcore/RLP.cpp

```diff
@@ -112,8 +112,10 @@ RLPs RLP::toList(int _flags) const
         else
             return ret;
     }
-    for (auto const& i : *this)
-        ret.push_back(i);
+    for (auto it = begin(); it != end(); it++)
```

Comment:
Codacy found an issue: [Prefer prefix ++/-- operators for non-primitive types.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=6313922)